### PR TITLE
#1885 UI-Header & Vuetify Config Tweaks

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#1885

*Description of changes:*

* Restructured the UI file directory to root level.
* Implemented SBC Common components Header & Footer.
* Implemented an entity info header to display dynamic data from Name Request Response. **Currently Static**
* Added unit testing for entity info header.
* Updated configurations to fix Vuetify.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: bcrs-ui/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "hello-world",
+  "name": "vue-frontEnd-Framework",
```
Comment (severinbeauvais) [inline bcrs-ui/package.json]: This should actually be this project's name (though it should be generic for the template project).

File: bcrs-ui/package.json [hunk 2]
```diff
@@ -11,36 +11,36 @@
   "dependencies": {
     "@mdi/font": "^4.5.95",
     "axios": "^0.18.0",
-    "core-js": "^3.4.3",
+    "core-js": "^3.4.4",
     "css-loader": "^0.28.11",
     "keycloak-js": "^6.0.0",
```
Comment (severinbeauvais) [inline bcrs-ui/package.json]: Don't need 'keycloak.js' (as it's now imported by sbc-common-components).

File: bcrs-ui/src/plugins/vuetify.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue'
```
Comment (severinbeauvais) [inline bcrs-ui/src/plugins/vuetify.js]: This is odd: I see another file with the same name and content below !?!?

File: bcrs-ui/package.json (hunk 1 above)
Comment (cameron-freshworks) [reply bcrs-ui/package.json]: Good point! I will rename this, and i'm really glad you mentioned the template as i will have to PR these same updates to the Entities repo where we are keeping a snapshot. Moving forward after this i think it makes sense to personalize to the app ( like you suggested! )

File: bcrs-ui/package.json (hunk 2 above)
Comment (cameron-freshworks) [reply bcrs-ui/package.json]: Thank you, will remove!

File: bcrs-ui/src/plugins/vuetify.js (hunk 3 above)
Comment (cameron-freshworks) [reply bcrs-ui/src/plugins/vuetify.js]: Very odd! I wonder how the .js file was created. I removed it.

**Comment by cameron-freshworks** [comment]
@severinbeauvais  Updated, ready for re-review 👍

File: src/components/common/Header.vue [hunk 4]
```diff
@@ -0,0 +1,73 @@
+<template>
```
Comment (severinbeauvais) [inline src/components/common/Header.vue]: Didn't we think we should use the header from sbc-common-components?

Comment (cameron-freshworks) [reply src/components/common/Header.vue]: You are correct sir, I implemented the common components but forgot to remove this placeholder. Will remove!

**Comment by cameron-freshworks** [comment]
> Looks great!

Thanks!

**Comment by cameron-freshworks** [comment]
Two approvals, merging!

**Pull request merged**